### PR TITLE
New charter

### PR DESCRIPTION
**Pull request by libremente**
This PR:
* introduces the first draft of the new charter
* introduces a "conventions" file in order to clarify some naming conventions

Let's work on the new structure of the file for clarity purposes first then I will fix it to style it as before.

### CODE REVIEW

File: CHARTER.md [hunk 1]
```diff
@@ -5,15 +5,85 @@
...
-|**5. Relationship to Other Projects**  |The ODF Advocacy Open Project can be related to other open source projects using ODF as their native file format such as Calligra, AbiWord and Gnumeric, or compatible with ODF such as Scribus.<br/>The ODF Advocacy Open Project can also be related to proprietary applications supporting ODF, provided they do not detract from the project’s original objective to promote the use of ODF.|
-|**6. Repositories and Licenses** |https://github.com/ODF-Advocacy Documents will be released under CC license Attribution 4.0 International (CC BY 4.0). |
-|**7. Initial Contributions from Existing Work**  |  |
+| **2. Abstract**   | 
+The objective of the PublicCode.yml Open Project is to
+develop the file format for describing public software, create awareness and
+educate the world about the benefits of using open source software in
+governments. The group will maintain the file format and keep it aligned with
```
Comment (libremente) [inline CHARTER.md]: Question: do we want to keep the scope of the `publiccode.yml` file centered around governments?

File: CHARTER.md [hunk 2]
```diff
@@ -5,15 +5,85 @@
...
+the needs of the governments and the agencies. |
+| **3. Purpose and Scope**  | 
+Public software is becoming more and more relevant each day, however its
```
Comment (libremente) [inline CHARTER.md]: Personally, I don't like the fact that we are referring to the same thing using a set of different words like "public code", "public software" etc. I believe we should focus on defining what we want to express and how. See the CONVENTIONS.md file for this reason.

File: CHARTER.md [hunk 3]
```diff
@@ -5,15 +5,85 @@
...
 
-For more information, please see:
-- [[Open Projects Guide]](https://github.com/oasis-open-projects/documentation/tree/master/guides)
-- [[Getting Started with Open Projects]](https://github.com/oasis-open-projects/documentation/blob/master/guides/getting-started-guide.md)
-- [[Open Project Rules]](https://www.oasis-open.org/policies-guidelines/open-projects-process)
-- [[Open Projects Website]](http://oasis-open-projects.org)
+<br/><br/>The following is an initial list of activities and materials
+which may be developed by the publiccode.yml Open Project working group:<br/><br/>
+<ol>
+    <li>**Standard Development
```
Comment (libremente) [inline CHARTER.md:32]: I believe this (Standard development) should be the real deal. We should clearly express this intention.

File: CHARTER.md [hunk 4]
```diff
@@ -5,15 +5,85 @@
...
+            <li>Definition of recognized rules for Country customization</li>
+            <li>Definition and development of working reference materials</li>
+    <li>**Tools**
+        <ol>
+            <li>Definition of a set of working tools</li>
+            <li>Development of a set of tools</li>
+    <li>**Awareness**
+        <ol>
+            <li>Dedicated Website</li>
+            <li>Media Outreach Campaign</li>
```
Comment (libremente) [inline CHARTER.md:44]: I am not entirely sure about this.

File: CHARTER.md [hunk 5]
```diff
@@ -5,15 +5,85 @@
...
+            <li>Relationship between public code and open source licenses</li>
+            <li>Importance of having a standard for public code</li>
+            <li>Advantages of the standard</li>
+            <li>Public code in governments</li>
+        </ol>
+    </li>
+    <li>**Marketing**
+        <ol>
+            <li>Impact of public code in an open government perspective</li>
+            <li>Economic value of reuse public software</li>
```
Comment (libremente) [inline CHARTER.md]: I am not sure of this either, but it may be of great impact if carried out involving other govs.

File: CHARTER.md [hunk 6]
```diff
@@ -5,15 +5,85 @@
...
+            <li>Standards & Innovation</li>
+        </ol>
+    </li>
+</ol>|
+| **4. Business Benefits**   |
```
Comment (libremente) [inline CHARTER.md:66]: Review this chapter. Does not communicate much.

File: CHARTER.md [hunk 7]
```diff
@@ -5,15 +5,85 @@
...
+</ol>|
+| **4. Business Benefits**   | 
+Government bodies and agencies are starting to adopt free and open source
+software thanks to a set of federal and national policies. As such, there is
+a strong need to clearly define a metadata specification standard to make such
+software easy indexable and eventually discoverable. The Publiccode.yml Open
+Project will also focus on studying the economic impact of the standard and how
+its introduction will affect the discoverabily and reusability process.
+
+| **5. Relationship to Other Projects**  |
```
Comment (libremente) [inline CHARTER.md:74]: Any clues about this? We have to insert other Open Projects or general existing projects (such as AppStream)?

**Review by libremente — COMMENTED** [review]
Added a few notes on the current version @ruphy @biancini. I'd appreciate your hints on this 😄

File: CHARTER.md (hunk 1 above)
Comment (biancini) [reply CHARTER.md]: Philosophically speaking I prefer "public goods/public adminstrations" to "governament".
It seems to me that the "public -thing-" better communicate the participative and open approach than "government".
_(my two cents)_

File: CHARTER.md (hunk 2 above)
Comment (biancini) [reply CHARTER.md]: I agree we shoudl be consistent. Should the `publiccode` be focussed on code?
I suggest we use "public code" over "public software", for instance...

File: CHARTER.md (hunk 5 above)
Comment (biancini) [reply CHARTER.md]: What about "Societal impact"?
I'm thinking about transparency of code (and thus of PAs in general) and the cultural approach by which public goods should be publicly managed and developed.
Is something we could specify here as an additional point?

File: CHARTER.md (hunk 1 above)
Comment (libremente) [reply CHARTER.md]: Yes, I think the same. Let's change this and then we'll see.

File: CHARTER.md (hunk 7 above)
Comment (libremente) [reply CHARTER.md:74]: @ruphy ^^

Comment (ruphy) [reply CHARTER.md:74]: great question, I'll ask today to be sure

**Pull request merged**